localization/am.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<localizationPack name="Armenia" version="1.0">
+	<currencies>
+		<currency name="Armenian dram" iso_code="AMD" iso_code_num="051" sign="֏" blank="1" conversion_rate="0.0021" format="2" decimals="0" />
```

Comment:
```suggestion
		<currency name="Armenian dram" iso_code="AMD" iso_code_num="051" blank="1" conversion_rate="0.0021" format="2" decimals="0" />
```

The sign is not anymore used. The symbol comes from CLDR data.

Reply:
May be will be better to replace the sign with "դրամ", it's in Armenian and means the same.